tests/specs/local-changes.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var test = require('tape')
+
+var Store = require('../../')
+var options = process.browser ? {
+  adapter: 'memory'
+} : {
+  db: require('memdown')
+}
+
+test('has "unsynced-local-docs" methods', function (t) {
+  t.plan(1)
+
+  var store = new Store('test-db-unsynced-local-docs', options)
+
+  t.is(typeof store.unsyncedLocalDocs, 'function', 'has "unsyncedLocalDocs" method')
+})
+
+test('creates correct "remote"', function (t) {
+  t.plan(1)
+
+  var dbName = 'test-db'
+  var store = new Store(dbName, options)
+
+  t.is(store.db.remote, dbName + '-remote', 'has correct "remote" name')
+})
+
+test('returns docs with "id" prop', function (t) {
+  t.plan(4)
+
+  var store = new Store('test-db-ids', options)
+
+  var localObj1 = {id: 'test1', foo: 'bar1'}
+  var localObj2 = {id: 'test2', foo: 'bar2'}
+
+  store.db.bulkDocs([localObj1, localObj2])
```

Comment:
I'd add `store.add` here. If you want to use db.bulkDocs, I think you need to set the ID properties to `_id`. But in the context of pouchdb-hoodie-store, we can just use `store.add`